examples/6tisch/6p-packet/sixp-node.c

```diff
@@ -49,7 +51,7 @@ PROCESS_THREAD(sixp_node_process, ev, data)
 
   sixtop_add_sf(&test_sf);
 
-  if(node_id == COORDINATOR_NODE_ID) {
+  if(linkaddr_cmp(&coordinator_addr, &linkaddr_node_addr)) {
```

Comment:
Are you sure this keeps the original intent? I mean, `node_id` can equal to `1` for other MAC addresses, not only this one.

Reply:
Yes I'd say the original intent was just to hardcode a coordinator.